marklogic-data-hub/src/main/java/com/marklogic/hub/dhs/installer/command/CanInstallDhsCommand.java

```diff
@@ -0,0 +1,35 @@
+package com.marklogic.hub.dhs.installer.command;
+
+import com.beust.jcommander.Parameters;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.marklogic.hub.dhs.installer.Options;
+import com.marklogic.hub.impl.Versions;
+import com.marklogic.hub.impl.Versions.MarkLogicVersion;
+import org.springframework.context.ApplicationContext;
+
+@Parameters(commandDescription = "Verify if current version of DHF can be installed in a DHS environment")
+public class CanInstallDhsCommand extends AbstractInstallerCommand {
+
+    ObjectMapper mapper = new ObjectMapper();
+
+    @Override
+    public void run(ApplicationContext context, Options options) {
```

Comment:
As a sanity check, I think all of what this method does should be put into a try/catch. And if any exception occurs, when create an ObjectNode with canBeInstalled=false and message = ex.getMessage(), and write that to stdout (which you can do with just node.toString(), I don't think we need to worry about pretty-printing here - or at least, we should write it in a way that avoids the chance of a JsonProcessingException).